api/oauth.go

```diff
@@ -0,0 +1,8 @@
+package api
+
+// OAuthToken is the API message for OAuthToken.
```

Comment:
Is there a particular reason that we have separate oauth.go file instead of placing it in autho.go?

Reply:
I guess auth if more of an internal service, and oauth on the other hands is more of an external interface.
There are tow different things.